recipes/scikit-time/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scikit-time" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: adb3199bb9084689999196185143027d4ef57e5eaee97ed6bf7c9a4193cb702e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py27 or osx or win] 
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - cython
+    - numpy
+    - scipy
+    - pip
+    - pybind11
+  run:
+    - python
+    - msmtools
+    - {{ pin_compatible('numpy') }}
+    - scipy
+    - pint
+    - scikit-learn
+
+test:
+  imports:
+    - sktime 
+
+about:
+  home: https://github.com/scikit-time/scikit-time
+  license: LGPL 
+  license_file: LICENSE.txt
+  summary: 'Estimate and analyze Markov processes based on time series data. '
+
+  #doc_url: https://notyet.there.to/
```

Comment:
Remove this comment unless you want to keep it.

Reply:
I'd like to keep it, so I won't forget to add the proper site once it is there.